docs/meshstack.openshift.index.md

```diff
@@ -5,62 +5,29 @@ title: Integration
 
 meshStack supports management of RedHat OpenShift platforms. OpenShift has a [Kubernetes](meshstack.kubernetes.index.md) core and provides additional services. It is available in both Open Source flavors (OKD) as well as enterprise offerings by RedHat.
 
-meshStack supports project creation, configuration, user management and SSO for OpenShift.
+meshStack supports project creation, configuration and user management for OpenShift.
 
 ## Integration Overview
 
 To enable integration with OpenShift, operators deploy and configure the meshStack OpenShift Connector. Operators can configure one or multiple `PlatformInstance`s of `PlatformType` OpenShift. This makes OpenShift available to meshProjects like any other cloud platform in meshStack.
 
-meshStack automatically configures OpenShift Projects and Permissions to integrate SSO with [meshStack Identity Federation](./meshstack.identity-federation.md).
-
 ## Prerequisites
 
 ### OpenShift Versions
 
-meshStack currently supports OpenShift version 3.7 as either Open-Source (OKD) or OpenShift Enterprise variants.
+meshStack currently supports OpenShift version 3.9+ as either Open-Source (OKD) or OpenShift Enterprise variants.
 
 ### IdP Configuration
 
-In order to integrate with [meshStack Identity Federation](./meshstack.identity-federation.md), operators need to configure the meshStack Identity Broker as an [OpenID Identity Provider in OpenShift](https://docs.okd.io/latest/authentication/identity_providers/configuring-oidc-identity-provider.html) using the following settings:
-
-```yml
-identityProviders:
-  - name: meshfed_oidc
-    challenge: false
-    login: true
-    mappingMethod: claim
-    provider:
-      apiVersion: v1
-      kind: OpenIDIdentityProvider
-      clientID: meshfed-oidc
-      clientSecret: ignored
-      extraScopes:
-      - email
-      - profile
-      - skipIdentifierSelection
-      extraAuthorizeParameters:
-        include_granted_scopes: "true"
-      claims:
-        id:
-        - sub
-        preferredUsername:
-        - sub
-        name:
-        - preferred_username
-        email:
-        - email
-      urls:
-        authorize: https://sso.example.meshcloud.io/auth/realms/meshfed/protocol/openid-connect/auth?response_mode=query
-        token: https://sso.example.meshcloud.io/auth/realms/meshfed/protocol/openid-connect/token
-        userInfo: https://sso.example.meshcloud.io/auth/realms/meshfed/protocol/openid-connect/userinfo
-```
+The same external IdP as configured for meshStack must be used for OpenShift (see [Identity Federation](meshstack.identity-federation.md#externally-provisioned-identities)).
+meshStack will identify and assign users in OpenShift via their euid as described in [Identity Federation](meshstack.identity-federation.md#externally-provisioned-identities).
```

Comment:
d: Maybe clarify that euid is an "external user ID" as someone who is only reading this section might not have heard of it before.

Reply:
added it in braces